src/cmd/link/internal/ld/pe.go

```diff
@@ -1372,6 +1387,9 @@ func (rt *peBaseRelocTable) addentry(ctxt *Link, s *sym.Symbol, r *sym.Reloc) {
 		Exitf("unsupported relocation size %d\n", r.Siz)
 	case 4:
 		e.typeOff |= uint16(IMAGE_REL_BASED_HIGHLOW << 12)
+	case 8:
+		// Todo(ragav): This case is a placeholder for now until we add relocation support.
```

Comment:
Generally don't like seeing Todo in code, is there a corresponding task tracking this?

Google [style guidelines](https://google.github.io/styleguide/cppguide.html#TODO_Comments) say it should be TODO in all caps.

Reply:
Yes, getting RELOC to work is on the list.